src/main/java/envoy/client/ui/listcell/ChatControl.java

```diff
@@ -21,30 +22,30 @@
 public class ChatControl extends HBox {
 
 	/**
-	 * @param contact the contact that should be formatted
+	 * @param chat the chat to display
 	 * @since Envoy Client v0.1-beta
 	 */
-	public ChatControl(Contact contact, int unreadMessagesAmount) {
+	public ChatControl(Chat chat) {
 		// Container with contact name
 		final var	vBox		= new VBox();
-		final var nameLabel = new Label(contact.getName());
+		final var	nameLabel	= new Label(chat.getRecipient().getName());
 		nameLabel.setWrapText(true);
 		vBox.getChildren().add(nameLabel);
-		if (contact instanceof User) {
+		if (chat.getRecipient() instanceof User) {
 			// Online status
-			final var	user		= (User) contact;
+			final var	user		= (User) chat.getRecipient();
 			final var	statusLabel	= new Label(user.getStatus().toString());
 			statusLabel.getStyleClass().add(user.getStatus().toString().toLowerCase());
 			vBox.getChildren().add(statusLabel);
 		} else // Member count
-			vBox.getChildren().add(new Label(((Group) contact).getContacts().size() + " members"));
+			vBox.getChildren().add(new Label(((Group) chat.getRecipient()).getContacts().size() + " members"));
 				
 		getChildren().add(vBox);
-		if (unreadMessagesAmount != 0) {
+		if (chat.getUnreadAmount() != 0) {
 			Region spacing = new Region();
 			setHgrow(spacing, Priority.ALWAYS);
 			getChildren().add(spacing);
-			final var unreadMessagesLabel = new Label("" + unreadMessagesAmount);
+			final var unreadMessagesLabel = new Label(Integer.toString(chat.getUnreadAmount()));
```

Comment:
Is there any reason why you wouldn't just replace the previous version with the new function?
```suggestion
			final var unreadMessagesLabel = new Label("" + chat.getUnreadAmount());
```

Reply:
The current way it's implemented seems more sensible to me as it omits the `StringBuilder`, but both are fine I guess.